doc/tutorials/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM clearlinux:latest
+
+RUN swupd bundle-add os-clr-on-clr os-core-dev python3-basic
+RUN pip3 install kconfiglib
+WORKDIR /workspace
+RUN chmod 777 /workspace
```

Comment:
Why do we need this?

Reply:
The command `RUN chmod 777 /workspace` is unnecessary, I have remove it, thanks a lot.